x-pack/plugins/core_telemetry/constants.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const PLUGIN_ID = 'core_telemetry';
+export const VIS_TELEMETRY_TASK = 'vis_telemetry';
+export const VIS_USAGE_TYPE = 'visualization_types';
```

Comment:
Minor: I'd prefer if there was a comment on each of these explaining what it's for. PLUGIN_ID is pretty clear to me. But the other two aren't.